server/src/test/java/io/spine/examples/kanban/server/board/DefaultColumnsTest.java

```diff
@@ -43,8 +49,69 @@ class DefaultColumnsTest extends UtilityClassTest<DefaultColumns> {
     @DisplayName("convert an enum value to the Title Case")
     void nameForConvertsToTitleCase() {
         String expected = "To Do";
-        String actual = DefaultColumns.nameFor(BoardInit.DefaultColumn.TO_DO);
+        String actual = DefaultColumns.nameFor(DefaultColumn.TO_DO);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("produce commands to add 'To Do', 'In Progress', 'Review' and 'Done' columns")
+    void additionCommandsProducesRightCommands() {
+        BoardId board = BoardId.generate();
+        AddColumn toDo =
+                AddColumn.newBuilder()
+                         .setBoard(board)
+                         .setName("To Do")
+                         .setDesiredPosition(
+                                 ColumnPosition.newBuilder()
```

Comment:
Please extract this bit. It is repeated here and down below. 

Actually, there's two methods to extract. One producing the command, and another building positions. 

Besides, the indentation is too big, let's think of those reading the code.